README.md

```diff
@@ -80,6 +80,35 @@ mysql -h localhost -D jackal -u jackal -p < mysql.sql
 
 Your database is now ready to connect with jackal.
 
+### Using PostgreSQL
+
+Create a user and a database for that user:
+
+```sql
+CREATE ROLE jackal WITH LOGIN PASSWORD 'password';
+CREATE DATABASE jackal WITH OWNER jackal;
+```
+
+Run the postgres script file to create database schema. In jackal's root directory run:
+
+```sh
+psql --user tester --password -f sql/postgres.up.psql
```

Comment:
🤔 

Not sure if `tester` should be replaced by `jackal` in this line. Could you @robzon confirm?

Reply:
Yep that's a typo, fixed!